server/lorax_server/models/types.py

```diff
@@ -163,6 +163,7 @@ def from_pb(
             return_token_type_ids=True,
             truncation=True,
             max_length=max_truncation,
+            padding=True,
```

Comment:
Hmm, is padding necessary? My understanding is flash attention should not require padding.

Reply:
Also, we should be doing tokenization on the Rust side now, so you should have access to the tokenized data from the request payload (see `flash_causal_lm.py` implementation).

Reply:
you need to pad the input tokens since we are feeding them into an encoder layer after embedding that does them by batch and so we need (batch, seq_len, hidden) dims